mig-frankfurt/dataelementhub.rest
# #46 - Reconstruct export output and import input

**What's in the PR**
*  Move ExportInfo and ExportRequest to datatransfer directory
* Update StagingImport.json and StagingImport.xsd to validate stagedElement as object instead of array
* Mark import label and stagedElements array as required

**How to test manually**
* Check if import label and stagedElements are required to start an import
* Check if stagedElement is transfered as object and not array
* Check if empty stagedElements array is not excepted

Test with [#39](https://github.com/mig-frankfurt/dataelementhub.model/pull/39)
